shared/middleware/channels_api.rb

```diff
@@ -173,10 +173,11 @@ class ChannelsApi < Sinatra::Base
   #
   # DELETE /v3/channels/<channel-id>/abuse
   #
-  # Clear an abuse score. Admin only.
+  # Clear an abuse score. Requires reset_abuse permission
   #
   delete %r{/v3/channels/([^/]+)/abuse$} do |id|
-    not_authorized unless admin?
+    # UserPermission::RESET_ABUSE
+    not_authorized unless has_permission?('reset_abuse')
```

Comment:
UserPermission::RESET_ABUSE works on my server, but not in tests. I haven't been able to figure out how to get it to, and the amount of time I've spent on it is already not justified by benefit of using the constant instead of a string. Added the constant in a comment so that people grepping for this should still find it.